geocity/apps/django_wfs3/urls.py

```diff
@@ -3,13 +3,13 @@
 from .routers import WFS3Router
 
 wfs3_router = WFS3Router()
-wfs3_router.register(r"permits", api_views.SubmissionViewSet, "submissions")
+wfs3_router.register(r"submissions", api_views.SubmissionViewSet, "submissions")
 wfs3_router.register(
-    r"permits_point", api_views.SubmissionPointViewSet, "submissions_point"
+    r"submissions_point", api_views.SubmissionPointViewSet, "submissions_point"
 )
 wfs3_router.register(
-    r"permits_line", api_views.SubmissionLineViewSet, "submissions_line"
+    r"submissions_line", api_views.SubmissionLineViewSet, "submissions_line"
 )
 wfs3_router.register(
-    r"permits_poly", api_views.SubmissionPolyViewSet, "submissions_poly"
+    r"submissions_poly", api_views.SubmissionPolyViewSet, "submissions_poly"
```

Comment:
`poly` means nothing, why not `area`?

Reply:
@monodo I have no clue about this, could you answer so i can then merge teh PR?

Reply:
Good idea, but out of scope of the refactor

Reply:
Yes but we might as well take advantage of it since we rename the urls?

Reply:
`Poly ` is not ideal, but closer to OGC geometry types definition thus, not sure renaming to area is a great idea. Correct name would by `Polygon `or `MultiPolygon`

Reply:
Anyway, we'll not rename now, but potentially at the end of refactor if time remains

Reply:
Yes but in this case you should also rename line to polyline ;)